test/import/FSHImporter.Extension.test.ts

```diff
@@ -54,6 +55,66 @@ describe('FSHImporter', () => {
           endColumn: 48
         });
       });
+
+      it('should throw an error when parent is declared more than once', () => {
+        const input = `
+        Extension: SomeExtension
+        Parent: ParentExtension
+        Id: some-extension
+        Title: "Some Extension"
+        Description: "An extension on something"
+        Parent: ParentExtension
+        `;
+
+        expect(() => {
+          importText(input);
+        }).toThrow(DuplicateMetadataError);
```

Comment:
Unfortunately, if we don't throw errors any more, then this test becomes less interesting.  Instead of checking for a thrown error, it should check to ensure that the *first* value of the duplicated key is the one kept.

Once we've implemented the bit about being able to validate that an _error_ was logged, we can revisit the test and add in that logic.